### ISSUE TITLE
Loading a map view does not reset the layer checkboxes

### ISSUE THREAD

**Issue by geneva78** [Maproom/qmapshack#535]
### Describe the bug

Loading a map view with only some layers enabled creates a window that correctly renders only those layers, but in the Maps window, checkboxes are checked for all layers. This makes it hard to determine what layers are actually enabled. Besides, it is not possible to add another layer by just checking the corresponding checkbox, because it needs to be unchecked first.

### To Reproduce

1. Activate a map with multiple layers.
2. Disable a few layers by unchecking the corresponding checkboxes.
3. Store the map view.
4. Close the map view.
5. Reload the map view.
6. Inspect the checkboxes for the layers in the Maps window.

### Expected behavior

Only checkboxes for enabled layers should be checked.

### Desktop

 - OS: Linux Mint 20.3
 - QMapShack Version: 1.16.1 (AppImage)
 - Window Manager: Cinnamon

### Additional context

By the way, it would be nice if the enabledLayers property in the view file would be honored even for maps that are not active, so that when a map is activated later, initially only those layers are enabled. That way, a view could define a default set of layers for a map.

**Comment by mitxel-m** [comment]
I can confirm this.

I have checked it with Waymarkedtrails.tms:

I have saved a view with only two layers enabled from waymarkedtrails.tms 
The `.view` file is ok, and it stores which  layers I have enabled when the view was created

```
[map]
b0bae2c9702052634bbd3bb923ba42b5\enabledLayers=1, 4

```
Whwn I reload this view , the map view is ok and it  is rendering only enabled layers (1,4)  as expected.
However in the map properties ui, the layer list widget shows all layers with a check. 

![](https://i.ibb.co/Bq0pm5w/pantallazo-033.jpg)

It seems to mark all layers with a check by default, regardless of what is saved in the .view file and what is rendered.

This only happens when I load the view file from the menu  (Archive - Load Map View).

When I restart QMS with that view already loaded, the layer list widget shows the layer checks correctly. 

---
Attached Waymarkedtrails.tms for testing:
(zipped)
[Waymarkedtrails.tms.zip](https://github.com/Maproom/qmapshack/files/9174147/Waymarkedtrails.tms.zip)
